tsfresh/feature_extraction/extraction.py

```diff
@@ -233,7 +233,7 @@ def _do_extraction(df, column_id, column_value, column_kind, column_sort,
 
     if distributor is None:
         if isinstance(data, Iterable):
-            if n_jobs == 0:
+            if n_jobs <= 1:
```

Comment:
I think that if statement is too loose, for `n_jobs=-1` you want to use all cores.

Reply:
Excellent point! I fixed it now.